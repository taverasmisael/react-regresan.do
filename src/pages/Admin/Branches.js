import React, { PureComponent } from 'react'

export default class Branches extends PureComponent {
  render() {
    return (
      <React.Fragment>
        Branches
      </React.Fragment>
    )
  }
}
