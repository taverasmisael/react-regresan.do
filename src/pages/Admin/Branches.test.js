import React from 'react'
import { shallow } from 'enzyme'
import Branches from './Branches'

describe('routes', () => {
  describe('Pages/Admin/Branches', () => {
    it('renders without crashing', () => {
      shallow(<Branches />)
    })
    it('matches snapshot', () => {
      expect(shallow(<Branches />)).toMatchSnapshot()
    })
  })
})
