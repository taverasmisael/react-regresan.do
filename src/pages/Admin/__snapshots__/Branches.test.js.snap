// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes Pages/Admin/Branches matches snapshot 1`] = `
<React.Fragment>
  Branches
</React.Fragment>
`;
